tozny/resource_realm_role.go

```diff
@@ -63,6 +63,31 @@ func resourceRealmRole() *schema.Resource {
 				Computed:    true,
 				ForceNew:    true,
 			},
+			"attribute": {
+				Description: "Arbitrary string-string key value pairs.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Elem: &schema.Resource{
```

Comment:
if an attribute is given do we need to recreate? then we need to re add all users, can we not use update realm role?